react/basic-example/src/components/Chatbox.js

```diff
@@ -0,0 +1,39 @@
+import React, { useContext, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { TalkSessionContext } from '../contexts/TalkSessionContext';
+
+/* 
+This is an example component that uses one of TalkJS' pre-built UI modes - or more specifically, the 'Chatbox' UI mode
+The Chatbox UI is typically used when the chat may be related to a speicific 'topic' such as a video, product order etc.
+*/
+function Chatbox(props) {
+
+    const chatContainer = useRef();
+    const {me, talkSession} = useContext(TalkSessionContext);
```

Comment:
i'm wondering, given that `me` is a Talk.User object, and we don't call it `talkMe`, maybe we should simply rename `talkSession` to `session` as well. Given that this component encapsulates a TalkJS thing, I'm not sure the prefix adds much.

Reply:
I think we often refer to talkSession elsewhere (generally in code examples etc., so IMO I'd rather keep this as is for now).

Reply:
true, but that's usually in the context of a global variable in vanilla JS environments (ie `window.talkSession`). for a local variable, scoped to a TalkJS-specific component, i wouldn't prefix it like that.

Reply:
I renamed it

Reply:
i think you didn't rename it everywhere, i checked out the latest version and the chat wouldn't load. I renamed `talkSession` to `session` on this line here in Chatbox.js and that fixed it.